src/plone/recipe/zope2instance/recipe.py

```diff
@@ -815,6 +815,7 @@ def build_wsgi_ini(self):
             'listen_port': '8080',
             # /todo
             'location': options['location'],
+            'max_request_body_size': options.get('max_request_body_size', '1073741824'),
```

Comment:
```suggestion
            'max_request_body_size': options.get(
                'max_request_body_size', '1073741824'),
```

This line seems to be too long according to the flake8 check.